src/main/java/seedu/taassist/logic/commands/DeletesCommand.java

```diff
@@ -19,14 +19,14 @@ public class DeletesCommand extends Command {
 
     public static final String COMMAND_WORD = "deletes";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Delete sessions for a class. "
+    public static final String MESSAGE_USAGE = "> Deletes sessions for a class.\n"
```

Comment:
I believe the original version is kept to maintain consistency for all the commands. Perhaps we can decide on the actual formatting in v1.4.